### PR TITLE
15_2_5 Entered num evaluation 

### PR DESCRIPTION
**Pull request by sergeroshe**
…dden one. Added entered num evaluation based consts, amended main program while loop with conditions.

### CODE REVIEW

File: 15_mini_project_work/2_num_guessing_game.py [hunk 1]
```diff
@@ -33,20 +36,26 @@ def main():
...
             print(ERROR_MESSAGE)
         else:
             guess_num = int(guess)
-            is_guess_wrong = False
+            if guess_num < rand_num:
+                print(TOO_SMALL_MESSAGE)
+            elif guess_num > rand_num:
+                print(TOO_BIG_MESSAGE)
+            else:
+                print(f'{WIN_MESSAGE} \n')
```
Comment (YanumVlad) [inline 15_mini_project_work/2_num_guessing_game.py:51]: Use print optional params instead

**Review by YanumVlad — CHANGES_REQUESTED** [review]

**Comment by YanumVlad** [comment]
@sergeroshe , task screenshot?

File: 15_mini_project_work/2_num_guessing_game.py [hunk 2]
```diff
@@ -33,20 +36,26 @@ def main():
     print(GREETING)
 
     is_guess_wrong = True
+    rand_num = random.randint(RANDOM_LEFT_BORDER, RANDOM_RIGHT_BORDER)
```
Comment (YanumVlad) [inline 15_mini_project_work/2_num_guessing_game.py:39]: bad name

Comment (sergeroshe) [reply 15_mini_project_work/2_num_guessing_game.py:39]: Which one?

Comment (sergeroshe) [reply 15_mini_project_work/2_num_guessing_game.py:39]: image

Comment (sergeroshe) [reply 15_mini_project_work/2_num_guessing_game.py:39]: Still unclear. Which one/ones of const/var names is/are bad?

**Review by YanumVlad — APPROVED** [review]

**Pull request merged**